cypress/integration/ix-video/data-setup.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  BIG_PLAY_BUTTON_SELECTOR,
+  MUTE_BUTTON_SELECTOR,
+  PLAYBACK_RATE_BUTTON_SELECTOR,
+  PLAYER_WITH_DATA_SETUP,
+  PLAY_BUTTON_SELECTOR,
+} from '../../fixtures/selectors';
+
+context('ix-video: data-setup attribute', () => {
+  before(() => {
+    cy.visit('/data-setup.html');
+  });
+
+  describe('with data-setup attribute', () => {
+    const host = PLAYER_WITH_DATA_SETUP;
+    const player = `${host} video`;
+    const bigPLayButton = `${host} ${BIG_PLAY_BUTTON_SELECTOR}`;
+    const playButton = `${host} ${PLAY_BUTTON_SELECTOR}`;
+    const muteButton = `${host} ${MUTE_BUTTON_SELECTOR}`;
+    const playbackRateButton = `${host} ${PLAYBACK_RATE_BUTTON_SELECTOR} button`;
+    const playbackRateValue = `${host} ${PLAYBACK_RATE_BUTTON_SELECTOR} > .vjs-playback-rate-value`;
+
+    it('should render', () => {
+      cy.get(host).should('exist');
+    });
+
+    it('should have a video element', () => {
+      cy.get(player).should('exist');
+    });
+
+    it('should display video controls', () => {
+      cy.get(bigPLayButton).should('exist');
+    });
+
+    it('should display large play button on first render', () => {
+      cy.get(bigPLayButton).should('exist');
+    });
+
```

Comment:
Just FYI, these two seem to be testing the same thing.

Reply:
💯 , meant to remove one from both of those tests apols.